Calculator/Calculator/CalculatorItem.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  CalculatorItem.swift
+//  Calculator
+//
+//  Created by Whales on 2023/05/31.
+//
+
+import UIKit
+
+protocol CalculatorItem {
+    
+}
```

Comment:
일반적으로 비어있는 프로토콜이나 타입, 익스텐션은 한줄로 줄여주는게 좋긴합니다. ㅎㅎ 꼭 이게 맞다는 아니니 참고만 해주세요~
```suggestion
protocol CalculatorItem { }
```

Reply:
오!! 사실 개행을 해놓으면서 뭔가 되게 비어있는 느낌이라 많이 허전했는데
저렇게 한 줄로 적어놓으니 완전 깔끔하네요!!
감사합니다 원비 🤩